scenarios/kubernetes_build.py

```diff
@@ -41,6 +73,13 @@ def main(args):
         print >>sys.stderr, (
             'Scenario should only run from either kubernetes or federation directory!')
         sys.exit(1)
+
+    # pre-check if target build exists in gcs bucket or not
+    # if so, don't make duplicated builds
+    if check_build_exists(args.release):
+        print >>sys.stderr, 'build already exists, exit'
+        sys.exit(1)
```

Comment:
do we want to fail builds? isn't that exactly the same problem we had before we allowed duplicate uploads?

Reply:
This should definitely be `sys.exit(0)`

Reply:
good catch!

Reply:
done